src/ring_theory/ideal/operations.lean

```diff
@@ -95,6 +95,21 @@ begin
   exact Hb _ hi _ hj,
 end
 
+/-- Dependent version of smul_induction_on -/
```

Comment:
Can you use the full lemma name in backticks so that it links?

Reply:
Done.
Does it need the prefix as in `submodule.smul_induction_on`?